contracts/ICO/EasyMineIco.sol

```diff
@@ -70,30 +70,22 @@ contract EasyMineIco {
   }
 
   modifier atStage(Stages _stage) {
-    if (stage != _stage) {
-      revert();
-    }
+    require(stage == _stage);
     _;
   }
 
   modifier isOwner() {
-    if (msg.sender != owner) {
-      revert();
-    }
+    require(msg.sender == owner);
     _;
   }
 
   modifier isPreIco() {
-    if (msg.sender != preIcoAddress) {
-      revert();
-    }
+    require(msg.sender == preIcoAddress);
     _;
   }
 
   modifier isValidPayload() {
-    if (msg.data.length != 0 && msg.data.length != 4 && msg.data.length != 36) {
```

Comment:
Perhaps change to isValidPayload(uint length) {
require(msg.data.length == length)
}